Paralayout/UIViewAdditions_Alignment.swift

```diff
@@ -296,21 +311,22 @@ public extension UIView {
 }
 
 
-extension UILabel: ContentPositionInset {
+extension UILabel: AlignmentPositionAdjusting {
     
     /// Adoption of the `CustomPosition` protocol for UILabels, insetting the top and bottom coordinates based on the label's font metrics.
-    public var contentPositionInsets: UIEdgeInsets {
+    public var alignmentPositionInsets: UIEdgeInsets {
         let capInsets = font.labelCapInsets(in: self)
         return UIEdgeInsets(top: capInsets.top, left: 0, bottom: capInsets.bottom, right: 0)
     }
     
     /// The size that fits the label's text in `.wrap` mode,
     /// - parameter width: the width to fit the text, passed through to `frameSize(thatFits:)`.
+    /// - parameter margins: An additional inset from the supplied width to use (optional, defaults to `0`).
```

Comment:
Should be specific in the comment that this is the margin on *each* side, not the total inset.